docs/guides/product-vocabulary.mdx

```diff
@@ -388,31 +387,20 @@ TBD
 
 ### help center / help docs / help documentation / learn more
 
-Use Help Center exclusively when linking to additional help documentation.
+Use Help Center exclusively when linking to additional help documentation, and
+link to the related document directly when possible.
 
 | **✅ Do **                                                                                | **❌ Don't**                                                                                |
 | ----------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
 | Learn how disputes work and how to prevent them in our [Help Center](#concepts-and-items) | Learn how disputes work and how to prevent them in our [Documentation](#concepts-and-items) |
-|                                                                                           | Want to know how disputes work and how to prevent them? [Learn more](#concepts-and-items)   |
-
-<br />
-
-### line items / product / service
-
-Use **line items** to describe the groupings of products and services on a
-quote, job or invoice.
 
-| **✅ Do**              | **❌ Don't**                    |
-| ---------------------- | ------------------------------- |
-| Add line item to quote | Add product or service to quote |
+Avoid using **learn more** as a link's label as it is poor for accessibility.
+Stick to descriptive labels that give more context of where the link is taking a
+user.
 
-Use **product / service** as labels for the name and description of individual
-line items, or when referring to the list of them available outside of an
-individual quote, job or invoice.
-
-| **✅ Do**                                                                                       | **❌ Don't**                                                                         |
-| ----------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------ |
-| Would you like to add recommended products and services based on your industry to your account? | Would you like to add recommended line items based on your industry to your account? |
+| **✅ Do **                                                                                | **❌ Don't**                                                                             |
+| ----------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| Learn how disputes work and how to prevent them in our [Help Center](#concepts-and-items) | ant to know how disputes work and how to prevent them? [Learn more](#concepts-and-items) |
```

Comment:
Typo, I'm guessing it's 'Want' instead of 'ant'

Reply:
@chris-at-jobber 👋

Reply:
Specs + builds fail on markdown for some silly hidden grammar stuff - @eddysims can you Take A Look?